server/jetty/src/main/java/io/deephaven/server/jetty/JettyConfig.java

```diff
@@ -128,6 +134,15 @@ public int port() {
     @Nullable
     public abstract Boolean http1();
 
+    public abstract OptionalInt http2StreamIdleTimeout();
+
+    /**
+     *
+     */
+    public int http2StreamIdleTimeoutOrDefault() {
+        return http2StreamIdleTimeout().orElse(0);
```

Comment:
Need to fix the Javadoc here. The default is defined here, but I intend to also define it in the default .prop file:
 * Defining it here prevents downstream breakage in applications built on top of DHC, while still leaving a reasonable default, without changing any Java or properties files
 * Defining it in the .prop file allows it to be passed to a client if so configured (I don't intend to enable that by default at this time), so that in theory if a deployment sets a concrete value here, clients can be informed at what time their streams will end if they've had no traffic.

Ideally we would solve this config situation differently, but I think this is a good compromise?

Reply:
Does 0 explicitly mean "don't ever idle timeout"?

Reply:
Any non-positive number is a legal value, and interpreted as "no timeout". Salient bit of Jetty:
```
        long streamIdleTimeout = getStreamIdleTimeout();
        if (streamIdleTimeout > 0)
            session.setStreamIdleTimeout(streamIdleTimeout);
            ```

Reply:
Thanks for clearing up that I was misreading this. I've updated the patch now to directly set our preferred timeout directly on each Http2 connection/session, rather than attempt to do it globally. I'll follow up with a bug report slash feature request to Jetty to see if this could be easier to use?

Reply:
Great.